rstudio-tidyverse/tox.ini

```
@@ -0,0 +1,57 @@
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+[tox]
+skipsdist = True
+skip_missing_interpreters = True
+envlist = pack, export-to-docker, sanity, integration
+
+[testenv]
+setenv =
+    PYTHONPATH={toxinidir}
+    PYTHONBREAKPOINT=ipdb.set_trace
+    CHARM_REPO=https://github.com/canonical/seldon-core-operator.git
+    CHARM_BRANCH=main
+    LOCAL_CHARM_DIR=charm_repo
+
+[testenv:pack]
+passenv = *
+allowlist_externals =
+    rockcraft
+commands =
+    rockcraft pack
+
+[testenv:export-to-docker]
+passenv = *
+allowlist_externals =
+    bash
+    skopeo
+    yq
+commands =
+    # export already packed rock to docker
+    bash -c 'NAME=$(yq eval .name rockcraft.yaml) && \
+             VERSION=$(yq eval .version rockcraft.yaml) && \
+             ARCH=$(yq eval ".platforms | keys | .[0]" rockcraft.yaml) && \
+             ROCK="$\{NAME\}_$\{VERSION\}_$\{ARCH\}.rock" && \
+             DOCKER_IMAGE=$NAME:$VERSION && \
+             echo "Exporting $ROCK to docker as $DOCKER_IMAGE" && \
+             skopeo --insecure-policy copy oci-archive:$ROCK docker-daemon:$DOCKER_IMAGE'
```

Let's use `rockcraft.skopeo` instead of `skopeo`. This reduces the dependencies used since we already need `rockcraft` to pack
```suggestion
    rockcraft
    yq
commands =
    # export already packed rock to docker
    bash -c 'NAME=$(yq eval .name rockcraft.yaml) && \
             VERSION=$(yq eval .version rockcraft.yaml) && \
             ARCH=$(yq eval ".platforms | keys | .[0]" rockcraft.yaml) && \
             ROCK="$\{NAME\}_$\{VERSION\}_$\{ARCH\}.rock" && \
             DOCKER_IMAGE=$NAME:$VERSION && \
             echo "Exporting $ROCK to docker as $DOCKER_IMAGE" && \
             rockcraft.skopeo --insecure-policy copy oci-archive:$ROCK docker-daemon:$DOCKER_IMAGE'
```